MIGRATING-0.6-0.7.md

```diff
@@ -113,6 +116,37 @@ TODO: Improve this section before release.
 - New fonts with `ascii` and `latin1` glyph subsets
 - `MonoFont` is now a struct instead of a trait
 
+The list of fonts has changed to the following:
+
+All fonts are provided in `embedded_graphics::mono_font::{ascii latin1}::[font name]`
```

Comment:
```suggestion
All fonts are provided in `embedded_graphics::mono_font::{ascii, latin1}::[font name]`
```

Reply:
Argh, thanks for catching this crap. Fixed!